### PR TITLE
Create HOME.html

### PR DESCRIPTION
**Pull request by Anthony-B-Russo10**
New, basic HTML design for the main page of Celestia.mobi.

### CODE REVIEW

**Comment by levinli303** [comment]
Please change the file name to index.html, also please fix the indentation. We indent with 2 spaces for the elements, like https://github.com/levinli303/celestia.mobi/blob/master/index.html

**Comment by levinli303** [comment]
And please also add a link to download Celestia
for now it is 
https://celestia.mobi/api/url

when we have celestia.space featuring mobile celestia, we can update the link to use the one in celestia.space

**Comment by Anthony-B-Russo10** [comment]
The indention has been updated and the link add. I had to name the file to index2.html due to it conflicting with the current index.html.

**Comment by levinli303** [comment]
well, you can keep it as index.html, as the current index.html is just temporary, by the way, the indentation is still incorrect, we need to ensure the <html> should be top-level, and I see in some cases where an inner element is indented by 1 space.

File: index2.html [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+    <head>
+      <meta charset='utf-8'>
+      <meta name='viewport' content='width=device-width'>
+      <title>Celestia.mobi</title>
+      <style> body { font-family: 'Helvetica Neue', Helvetica, Arial, sans-serif; padding:1em; } </style>
+    </head>
+    <body>
+      <h1>Celestia.mobi</h1> 
+        <p>
+          Celestia.mobi is the repostiory for all of the addons that are avaliable
```
Comment (levinli303) [inline index2.html:12]: please put text in one line

File: index2.html [hunk 2]
```diff
@@ -0,0 +1,77 @@
...
+				  for download for the mobile versions of Celestia. 
+	</p>
+      <h3>Download Link</h3>
+	<p>
+	  <a href="https://celestia.mobi/api/url">Download link</a>
+	</p>
+    </body>
```
Comment (levinli303) [inline index2.html:19]: body should end just before html closing tag

File: index2.html [hunk 3]
```diff
@@ -0,0 +1,77 @@
...
+	   <p>
+	     <a href="https://celestia.mobi/resources/category.html?id=0EEF518F-C9C8-E27D-AA9C-8747FF2094B7">Exoplanets</a>
+	   </p>
+	   <p>
+	     <a href="https://celestia.mobi/resources/category.html?id=3F7546F9-D225-5194-A228-C63281B5C6FD">Nebulae</a>
+	   </p>
+	   <p>
+	     <a href="https://celestia.mobi/resources/category.html?id=DEAF9C38-7E4B-7DFB-CA72-B496B1652550">Blackholes</a>
+	   </p>
+        </div>
```
Comment (levinli303) [inline index2.html:40]: wrong indentation

**Comment by Anthony-B-Russo10** [comment]
The first issue was resolved.

**Comment by levinli303** [comment]
I've updated index.html, and resources/index.html, please see to https://celestia.mobi

**Comment by Anthony-B-Russo10** [comment]
That fits the site better then what I had done. Though eventually a background image will have to be put in place on the site similar to the main site or the Forums.

**Pull request closed without merging**